### PR TITLE
Task/2

### PR DESCRIPTION
**Pull request by askomar**

### CODE REVIEW

File: codewars/515e271a311df0350d00000f.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+function squareSum(numbers) {
+  let result = 0;
+  numbers.forEach((num) => {
+    result += num * num;
+  });
+  return result;
+}
+
+console.log(squareSum([-8, -15]));
```
Comment (mTeeeur) [inline codewars/515e271a311df0350d00000f.js:16]: Look's good but we can improve this implementation using reduce method =)

File: codewars/5513795bd3fafb56c200049e.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+ */
+
+function countBy(x, n) {
+  let z = [];
+  z.push(x);
+  for (let i = 0; i < n - 1; i++) {
+    z.push(z[z.length - 1] + x);
+  }
+  return z;
+}
```
Comment (mTeeeur) [inline codewars/5513795bd3fafb56c200049e.js:19]: Missing newline a the ending of  file

File: codewars/5513795bd3fafb56c200049e.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+ * @link https://www.codewars.com/kata/5513795bd3fafb56c200049e
+ * @author Komarovsky Andrey (askomar)
+ *
+ * Task short description:
+ *  1. Create a function that will return an array of the first (n)
+ * multiples of (x)
+ */
+
+function countBy(x, n) {
+  let z = [];
```
Comment (mTeeeur) [inline codewars/5513795bd3fafb56c200049e.js:13]: Here we can using const instead of let

File: codewars/5583090cbe83f4fd8c000051.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+function digitize(n) {
+  return n
+    .toString()
+    .split("")
+    .reverse()
+    .map((n) => {
+      return Number(n);
+    });
+}
```
Comment (mTeeeur) [inline codewars/5583090cbe83f4fd8c000051.js:22]: Missing newline

File: codewars/55685cd7ad70877c23000102.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+/**
+ * @difficulty 8 kyu
+ * @description This file contains an answer for <Return Negative> codewars task
+ * @link https://www.codewars.com/kata/55685cd7ad70877c23000102
+ * @author Komarovsky Andrey (askomar)
+ */
+
+function makeNegative(num) {
+  return num < 0 ? num : -1 * num;
```
Comment (mTeeeur) [inline codewars/55685cd7ad70877c23000102.js:9]: We can return -num =)

File: codewars/5513795bd3fafb56c200049e.js [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+  z.push(x);
+  for (let i = 0; i < n - 1; i++) {
```
Comment (mTeeeur) [inline codewars/5513795bd3fafb56c200049e.js:15]: Try to avoid usage for loops =)

File: codewars/574b3b1599d8f897470018f6.js [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+> codewars task
+ * @link https://www.codewars.com/kata/574b3b1599d8f897470018f6
+ * @author Komarovsky Andrey (askomar)
+ *
+ * Task short description:
+ *  Write a fucntion that given a floor in the american system returns the 
+ * floor in the european system
+ */
+function getRealFloor(n) {
+  return n >= 1 && n >= 13 ? n - 2 : n >= 1 ? n - 1 : n == 0 ? 0 : n;
```
Comment (mTeeeur) [inline codewars/574b3b1599d8f897470018f6.js:14]: Hard to read the code if you using more than one ternary operator

File: codewars/5966e33c4e686b508700002d.js [hunk 8]
```diff
@@ -0,0 +1,10 @@
+/**
+ * @difficulty 8 kyu
+ * @description This file contains an answer for <Sum The Strings> codewars task
+ * @link https://www.codewars.com/kata/5966e33c4e686b508700002d
+ * @author Komarovsky Andrey (askomar)
+ */
+
+function sumStr(a, b) {
+  return `${(a == "" ? 0 : Number(a)) + (b == "" ? 0 : Number(b))}`;
```
Comment (mTeeeur) [inline codewars/5966e33c4e686b508700002d.js:9]: Why do you use == instead of ===?

File: codewars/596c6eb85b0f515834000049.js [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+but it is my own solution
+
+*/
+var replaceDots = function (str) {
+  let strCopy = str;
+  while (strCopy.includes(".")) {
+    strCopy = strCopy.replace(/\./, "-");
+  }
+  return strCopy;
+};
```
Comment (mTeeeur) [inline codewars/596c6eb85b0f515834000049.js:22]: Missing newline

**Review by mTeeeur — CHANGES_REQUESTED** [review]
Looks good but need to rework some solutions and create for each kata ticket and branch. You must follow git-flow guideline when creating new branches.

File: codewars/515e271a311df0350d00000f.js (hunk 1 above)
Comment (askomar) [reply codewars/515e271a311df0350d00000f.js:16]: I'm try to use a reduce methot but I have some trouble with it)

Comment (mTeeeur) [reply codewars/515e271a311df0350d00000f.js:16]: > I'm try to use a reduce methot but I have some trouble with it)

Try it one more time please =)

File: codewars/55685cd7ad70877c23000102.js (hunk 5 above)
Comment (askomar) [reply codewars/55685cd7ad70877c23000102.js:9]: Really) Oh Holy Jesus)

File: codewars/574b3b1599d8f897470018f6.js (hunk 7 above)
Comment (askomar) [reply codewars/574b3b1599d8f897470018f6.js:14]: Okey)

File: codewars/5966e33c4e686b508700002d.js (hunk 8 above)
Comment (askomar) [reply codewars/5966e33c4e686b508700002d.js:9]: It is my mistake, I must use strong operator

**Pull request closed without merging**